py/__init__.py

```diff
@@ -563,7 +563,8 @@ def updateTrace(self, X, Y, win, env=None, name=None,
             'opts': opts,
```

Comment:
Can you add a print up here in updateTrace that notes using updateTrace will be deprecated in the future and that users should switch to using `func(..., update='append', name=<trace_name>)`?

Reply:
Addressed.